tests/python/test_solana.py

```diff
@@ -1,65 +1,36 @@
 from ragger.backend import RaisePolicy
-from ragger.navigator import NavInsID, NavIns
 from ragger.utils import RAPDU
 
 from .apps.solana import SolanaClient, ErrorType
 from .apps.solana_cmd_builder import SystemInstructionTransfer, Message, verify_signature, OffchainMessage
 from .apps.solana_utils import FOREIGN_PUBLIC_KEY, FOREIGN_PUBLIC_KEY_2, AMOUNT, AMOUNT_2, SOL_PACKED_DERIVATION_PATH, SOL_PACKED_DERIVATION_PATH_2, ROOT_SCREENSHOT_PATH
-from .apps.solana_utils import enable_blind_signing, enable_short_public_key, enable_expert_mode, navigation_helper_confirm, navigation_helper_reject
+from .apps.solana_utils import enable_blind_signing, enable_short_public_key, enable_expert_mode
 
 
 class TestGetPublicKey:
 
-    def test_solana_get_public_key_ok(self, backend, navigator, test_name):
+    def test_solana_get_public_key_ok(self, backend, scenario_navigator, navigator, test_name):
```

Comment:
`navigator` parameter seems useless, no?

Reply:
Indeed. Removing it.